ebu_tt_live/documents/converters.py

```diff
@@ -18,13 +18,16 @@ def ebutt3_to_ebuttd(ebutt3_in, media_clock):
     :param ebutt3_in:
     :return:
     """
+    denester_node = DenesterNode(
```

Comment:
You don't need to make a node here, explicitly like this. Either keep the denester functionality in a Denester class and keep the code as it was before or take out the denesting altogether from the converter.

The argument for taking out the denesting from the EBUTT3 to EBUTTD converter is that if the deployment configuration is such that the documents are already denested then it is not needed, but if it is needed then the Denester node can be chained "in series" with the converter, to arrange for the input to the converter to be acceptable.